brainglobe_atlasapi/atlas_generation/atlas_scripts/mpin_zfish.py

```diff
@@ -1,9 +1,9 @@
 __version__ = "1"
```

Comment:
We should be consistent if we want to move the version away from here (but I'm not sure we do?)

Reply:
This is not finished. I'm working on finding the template and annotation files as it is not straightforward to download them. But yes, it will be consistent across scripts.

Reply:
OK! I've opened https://github.com/brainglobe/brainglobe-atlasapi/issues/423 to further think about what we want to do with `__version__` (our current use feels hacky to me anyway!).